pages/desktop/consumer_pages/base.py

```diff
@@ -27,11 +27,6 @@ def page_title(self):
     def wait_for_page_to_load(self):
         self.wait_for_element_not_present(*self._load_page_details_baloon_locator)
 
-    def scroll_to_element(self, *locator):
```

Comment:
I moved this into `page.py` and changed it to accept an element rather than a locator.